main.tf

```diff
@@ -0,0 +1,36 @@
+terraform {
+  required_version = ">= 0.11.7"
+  backend "gcs" {
+    bucket  = "insert bucket"
+    prefix  = "terraform/spotinst-kubernetes-controller"
+  }
+}
+
+provider "google" {
+  project = var.gce-project
+  region  = var.gce-region
+  version = "~> 2.9.1"
+}
+
+provider "helm" {
```

Comment:
Use helm provider >1.0
There is no need for tiller in helm3. Please cleanup code